napalm_logs/config/opengear/LOGOUT.yml

```
@@ -0,0 +1,13 @@
+#<30>ztp-tankstelle_1-ts port02 2018-Nov-12 10:47:34.318 LOGOUT: root
+
+messages:
+  - error: LOGOUT
```

```suggestion
  - error: USER_LOGOUT
```

Similar to the above, let's keep it consistent.